lib/ansible/runner/filter_plugins/core.py

```diff
@@ -76,9 +76,12 @@ def skipped(*a, **kw):
 
 def mandatory(a):
     ''' Make a variable mandatory '''
-    if not a:
+    try:
+        a
+    except NameError:
         raise errors.AnsibleFilterError('Mandatory variable not defined.')
```

Comment:
This will never happen: 'a' is a defined local variable, so Python will never raise NameError.

In effect the `mandatory` filter is now a no-op.